Automattic/woocommerce-payments
# Don't load the WooPay scripts when the WCPay gateway isn't available

Fixes #5118 

#### Changes proposed in this Pull Request

- Check whether WCPay is in the available payment gateways before enqueueing WooPay styles and scripts.
- Don't load the WooPay scripts when the WCPay gateway isn't available.

#### Testing instructions

1. Enable WooPay
2. Enable [Conditional Shipping and Payments](https://woocommerce.com/products/conditional-shipping-and-payments/)
3. Create a condition to disable WCPay on the checkout page
4. As a shopper, add a product to your cart and go to the checkout page. Make sure the condition you created to disable 5. WCPay is met
6. On the checkout page, notice that WCPay isn't displayed in the available payment gateways
7. Open the browser developer tools -> Network -> search for checkout.js and checkout.css
8. Confirm that WooPay scripts (platform-checkout.js) and styles (platform-checkout.css) are not loaded

<img width="1500" alt="image" src="https://user-images.githubusercontent.com/41606954/201482352-0a162f91-438c-4256-89fd-6474aefe6052.png">

Regressions:
1. Confirm you can still place an order via WooPay when WCPay is available.

-------------------


**Post merge**
